solr/bin/solr.cmd

```diff
@@ -994,7 +994,7 @@ set "EXAMPLE_DIR=%SOLR_TIP%\example"
 set TMP_SOLR_HOME=!SOLR_HOME:%EXAMPLE_DIR%=!
 IF NOT "%TMP_SOLR_HOME%"=="%SOLR_HOME%" (
   set "SOLR_LOGS_DIR=%SOLR_HOME%\..\logs"
-  set "LOG4J_CONFIG=file:///%SOLR_SERVER_DIR%\resources\log4j2.xml"
+  set "LOG4J_CONFIG=%SOLR_SERVER_DIR%\resources\log4j2.xml"
```

Comment:
You can't make URLs of things unless you encode them, if there are spaces in `SOLR_SERVER_DIR` this is all messed up. So i fixed it to use a standard path instead: https://logging.apache.org/log4j/log4j-2.1/faq.html#config_location

Looks like the same bug exists in `bin/solr` too, i'll make a space on os X and try it out.